io.catenax.software_information/1.0.0/SoftwareInformation.ttl

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
+# Copyright (c) 2023 German Edge Cloud GmbH & Co. KG
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 Mercedes Benz AG
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 Siemens AG
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:samm:io.catenax.sofware_information:1.0.0#>.
+
+:SoftwareInformation a samm:Aspect;
+   samm:preferredName "Software Information"@en;
+   samm:description "This aspect model describes information about one or more software of a part (e.g. software of an ECU)."@en;
+   samm:properties (:catenaXId :softwareInformation);
+   samm:operations ();
+   samm:events ().
+
+:catenaXId a samm:Property;
+   samm:preferredName "Catena-X ID"@en;
+   samm:description "The Catena-X ID of the given part (e.g. the assembly), valid for the Catena-X dataspace."@en;
+   samm:characteristic :CatenaXIdTraitCharacteristic;
+   samm:exampleValue "urn:uuid:454ffa8e-f88d-4ad1-be45-06981756aeb1".
+
+:softwareInformation a samm:Property;
+   samm:preferredName "Software Information"@en;
+   samm:description "Describes the sofware components with the most important data(e.g. name, version,..)."@en;
+   samm:characteristic :SetOfSoftwareCharacteristic.
+
+:CatenaXIdTraitCharacteristic a samm-c:Trait;
+   samm:description "Trait to ensure UUID v4 data format"@en;
+   samm-c:baseCharacteristic :Uuidv4Characteristic;
+   samm-c:constraint :Uuidv4RegularExpression.
+
+:SetOfSoftwareCharacteristic a samm-c:Set;
+   samm:preferredName "Set of Software"@en;
+   samm:description "Set of software that lists all software used by a hardware."@en;
+   samm:dataType :SoftwareData.
+
+:Uuidv4Characteristic a samm:Characteristic;
```

Comment:
there is a shared aspect model available for uuidv4. please check on using that, instead of declaring yourself in model.